dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/job/TrackerImportParamsSerdeTest.java

```diff
@@ -64,9 +65,9 @@ protected void initTest()
     {
     }
 
+    @SneakyThrows
```

Comment:
Even its name does not give me confidence in that I want to use it :joy: I know this is a test but I would not use it. I don't want sneaky code. Even if its annoying, I like it obvious 😋 Also its not in https://github.com/dhis2/wow-backend/blob/master/guides/lombok.md#allowed-annotations

Reply:
I used it just because the exception here is coming from a subdependency and it is messing up what I am doing.
But I agree.
Just using Exception instead